creation/interfaces-and-networks.md

```diff
@@ -736,6 +736,10 @@ spec:
 > **Note:** for some NICs (e.g. Mellanox), the kernel module needs to be
 > installed in the guest VM.
 
+> **Note:** in this example, dedicatedCpuPlacement is enabled,
+> but it can only be enabled if cpumanager is enabled on SRIOV capable workers.
```

Comment:
maybe it's better, to put it differently... something like: 

"Placement on dedicated CPUs can only be achieved if the Kubernetes CPU manager is running on the SR-IOV capable workers."

Reply:
Done.